test/python/docker/compat/test_containers.py

```diff
@@ -260,3 +262,24 @@ def test_mount_rw_by_default(self):
                 ctr.remove()
             if vol is not None:
                 vol.remove(force=True)
+
+    def test_wait_next_exit(self):
+        self.skipTest("Skip until fix container-selinux#196 is available.")
```

Comment:
Alternatively I could try to detect current version of `container-selinux` pkg and skip test conditionally.
I am not sure if it easy/worth.

Reply:
fix should be in `v2.194.0`

Reply:
@cevich could we update the CI images?

Reply:
Image update coming in https://github.com/containers/podman/pull/16692

Not sure if it includes any desired fixes but given a version number, I can check.

Reply:
Yuk. I'm sorry this became pythonized: (1) it will never get unskipped, because my fixme-check script doesn't check python, and (2) it will never get run anyway because gating tests only run the BATS tests, not python. @matejvasek could I ask you to submit a new PR with your system test as it was before?